mission_control/include/mission_control/mission_control.h

```diff
@@ -118,7 +121,6 @@ class MissionControlNode
 
   int m_current_mission_id;
   int m_mission_id_counter;
-  Mission* m_cur_mission;
 
   std::map<int, Mission*> m_mission_map;
```

Comment:
@patapufate consider using an `std::unordered_map`

Reply:
@patapufate consider using smart pointers to manage `Mission` lifetimes.

Reply:
Fixed